.gitignore

```diff
@@ -12,12 +12,6 @@ config.guess
 depcomp
 build
 config.log
-src/gpu/*.mrc
-src/gpu/*.o
-src/gpu/RemoteSystemsTempFiles
```

Comment:
Ignoring .o files actually seems like a good idea, doesn't it?

Reply:
These are left over from when I compiled the gpu core library separate (prior to) the full cisTEM compile. Nothing should show up in src/gpu with a proper build.